.npmignore

```diff
@@ -0,0 +1,7 @@
+*.iml
+.idea
+coverage
+.DS_Store
+Thumbs.db
+node_modules
+*.log
```

Comment:
Is this file necessary? The npm docs [say](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package) that if there's no `.npmignore` it'll use the `.gitignore` file instead.